gcloud/pubsub/subscription.py

```diff
@@ -74,9 +74,7 @@ def exists(self):
         """
         conn = self.topic.connection
         try:
-            conn.api_request(method='GET',
-                             path=self.path,
-                             query_params={'fields': 'name'})
```

Comment:
https://developers.google.com/apis-explorer/#p/pubsub/v1beta2/pubsub.projects.subscriptions.get

has fields and the fields editor lists name.

What in particular isn't supported?